resources/js/langman.js

```diff
@@ -80,9 +80,9 @@ new Vue({
          * Add a new translation key
          */
         addNewKey(key) {
```

Comment:
Sorry, i give you your honor, i just misswrite alert message in wrong action, this should be in addLanguage action, not into addNewKey... I will fix it soon.